vm-setup/roles/v1aX_integration_test/tasks/move.yml

```
@@ -165,6 +165,16 @@
     until: (target_running_pods is succeeded) and
            (target_running_pods.resources | length == 0)
 
+  - name: Check if all deployments replicas are available
+    kubernetes.core.k8s_info:
+      kind: Deployment
+      kubeconfig: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
+    register: ironic_bmo_deployments
```

This is all deployment, right?
```suggestion
    register: deployments
```